packages/server/lib/automation/cookies.ts

```diff
@@ -205,7 +219,7 @@ export class Cookies {
     })
   }
 
-  async clearCookies (data, automate) {
+  async clearCookies (data: AutomationCookie[], automate: AutomationMessageFn<AutomationCookie[], AutomationCookie[]>) {
```

Comment:
TODO: clear cookie sometimes passes `null` into normalizecookieprops. I wonder if we should handle that here to make the normalization FN a bit more strealined